src/v0/destinations/twitter_ads/transform.js

```diff
@@ -131,6 +131,20 @@ function processTrack(message, metadata, destination) {
     identifiers.push({ twclid: message.properties.twclid });
   }
 
+  if (message.properties.ip_address) {
```

Comment:
why can't we use https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/#contextual-fields:~:text=and%20type.-,ip,-String

Reply:
if we are mapping `ip_address` and `user_agent` from context, why are setting same values in properties too? @aanshi07

Reply:
As per the doc and implementation identifiers are required to be passed in properties.